src/Tribe/Meta/Chunker.php

```diff
@@ -93,19 +94,18 @@ public function hook() {
 	 * This will just fetch the keys for the supported post types, not the values.
 	 */
 	protected function prime_chunked_cache() {
-		/** @var wpdb $wpdb */
-		global $wpdb;
-		$query = $wpdb->prepare( "SELECT post_id, meta_key FROM {$wpdb->postmeta}
-			WHERE meta_key LIKE %s
-			AND meta_key NOT LIKE %s",
-			$this->meta_key_prefix . '%', $this->meta_key_prefix . '%_chunk'
-		);
-		$results = $wpdb->get_results( $query );
-
 		$this->chunks_cache = array();
-		foreach ( $results as $result ) {
-			$real_meta_key = str_replace( $this->meta_key_prefix, '', $result->meta_key );
-			$this->chunks_cache[ $this->get_key( $result->post_id, $real_meta_key ) ] = null;
+
+		$chunked_keys = get_option( $this->chunked_keys_option_name );
+
+		if ( empty( $chunked_keys ) ) {
+			return;
+		}
+
+		foreach ( $chunked_keys as $post_id => $keys ) {
+			foreach ( $keys as $key ) {
+				$this->chunks_cache[ $this->get_key( $post_id, $key ) ] = null;
```

Comment:
This test https://github.com/moderntribe/tribe-common/blob/9d8c8c75789d697ce91d2da1e8b89b86920f9956/tests/wpunit/Tribe/Meta/ChunkerTest.php#L281-L281 clarifies it.
If nothing is being tagged as chunkable in the context of the request the cache will never be primed and any meta chunked, and registered as chunkable, in a previous request would not be treated as such.